src/Instrumentation/Laravel/src/LaravelInstrumentation.php

```diff
@@ -13,7 +13,11 @@ class LaravelInstrumentation
 
     public static function register(): void
     {
-        $instrumentation = new CachedInstrumentation('io.opentelemetry.contrib.php.laravel');
+        $instrumentation = new CachedInstrumentation(
+            'io.opentelemetry.contrib.php.laravel',
+            null,
+            'https://opentelemetry.io/schemas/1.25.0'
```

Comment:
`1.24.0` is the max supported here.

Reply:
Apologies, re-reading that and I didn't give much info!

It's been discussed to aim to release `0.x` packages with a dependency on `php: ^8.0` (as auto-instrumentation extension works with that).

`sem-conv: 1.24.0` was the last version before the dependency in the package was bumped to `php: ^8.1`.

Hope this helps explain my previous comment 👍

Reply:
I had understood in [this comment](https://github.com/open-telemetry/opentelemetry-php/issues/1312#issuecomment-2118658228) that we should update to the latest semantic version which is currently 1.25.

For example, the contrib-auto-slim package has a version above `>= 1.0.0` and in this [PR ](https://github.com/open-telemetry/opentelemetry-php-contrib/pull/259) it was defined to use the URL schema in version 1.25, however in composer.json it has the `php: ^8.0`.

I was a little confused regarding whether the URL schema version should be defined based on the supported PHP versions according to their respective composer.json or go with the latest published version